source/agora/consensus/protocol/Nominator.d

```diff
@@ -1081,6 +1103,61 @@ extern(D):
             this.last_confirmed_height = Height(envelope.statement.slotIndex);
     }
 
+    /***************************************************************************
+
+        Get the UTXO hash for the index of the validator at a height
+
+        Params:
+            slot_idx = the height for which we search the index of the validator
+            index = the index of the validator
+
+        Returns:
+            the hash of the frozen UTXO
+
+    ***************************************************************************/
+
+    public StellarHash getNodeUTXO (uint64_t slot_index, uint64_t index)
+        @safe nothrow
+    {
+        Hash utxo;
+        if (slot_index == 1)
+        {
+            Block b0;
+            try
+            {
+                b0 = this.ledger.getBlocksFrom(Height(0))[0];
+            }
+            catch (Exception exc)
+            {
+                log.error("Exception happened with getting block at requested height: {}", exc);
+            }
+            assert(b0.header.enrollments.length > index);
+            utxo = b0.header.enrollments[index].utxo_key;
+        }
+        else
```

Comment:
Why is `slot_index == 1` treated specially ?

Reply:
Removed it.